test_files/v2/flows.json

```diff
@@ -22,7 +22,7 @@
             "archived": false,
             "labels": [],
             "expires": 720,
-            "created_on": "2016-01-06T15:33:00.813162Z",
+            "created_on": "2016-01-06T15:33:00.813162+00:00",
```

Comment:
We aren't changing this format actually, these will remain as Z, though we should support both.

Reply:
Yep we still support both